formats/finder/frontend/examples/esi-funds.json

```diff
@@ -0,0 +1,196 @@
+{
+  "base_path": "/european-structural-investment-funds",
+  "title": "European Structural and Investment Funds (ESIF)",
+  "description": "",
+  "format": "finder",
+  "need_ids": [],
+  "locale": "en",
+  "updated_at": "2015-04-10T13:29:08.695Z",
+  "public_updated_at": "2015-04-01T11:42:34.000+00:00",
+  "details": {
+    "beta": true,
+    "beta_message": null,
+    "document_noun": "call",
+    "email_signup_enabled": false,
+    "eu_logo_path": "European Structural and Investment Funds",
```

Comment:
thank you for picking this up! 
So... it will be a path, but we don't know what the path will be yet. I'll fix this asap the file will have been uploaded.